Yoast/javascript
# Add Flesch reading ease assessment for Portuguese

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements the Flesch reading ease assessment for Portuguese.
* [Yoast SEO Free] Adds the Flesch reading ease assessment for Portuguese.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* Team-internal testing
    * Add more specs in `countPortugueseSpec.js` and make sure the words that are tested have correct syllable count.
    * Add more specs in `calculateFleschReadingSpec.js` and delete them after testing
    * Run `yarn test` and make sure that everything pass.

## QA Test instructions 
    * In your WordPress testing environment, set the language to Portuguese.
    * Open a new post or page. 
    * Add Portuguese text
    * Make sure that the Flesch Reading score appears in the assessment bar.
## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-478
